nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>I don't think this import is needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Removed import as suggested.</p>